mangaki/mangaki/models.py

```diff
@@ -500,8 +591,15 @@ class WorkCluster(models.Model):
     merged_on = models.DateTimeField(blank=True, null=True)
     origin = models.ForeignKey(Suggestion, related_name='origin_suggestion', on_delete=models.CASCADE, blank=True, null=True)
 
+    def get_difficulty(self):
```

Comment:
Let's get a [`@cached_property`](https://docs.djangoproject.com/fr/2.0/ref/utils/#module-django.utils.functional) on this, I think.

Reply:
OK but I hope I won't break the rest of the code O:-) (should not put the `()` anymore).